frontend/src/components/TopContributorsList.tsx

```diff
@@ -44,32 +43,25 @@ const TopContributorsList = ({
         </div>
       }
     >
-      <div className="grid gap-4 sm:grid-cols-1 md:grid-cols-3">
+      <div className="grid gap-4 sm:grid-cols-1 md:grid-cols-4">
```

Comment:
One suggestion:
```suggestion
      <div className="grid gap-4 sm:grid-cols-1 md:grid-cols-3 lg:grid-cols-4">
```
Right now, on smaller screens the names are getting cut off a lot before we reach next break point. 
<img width="740" height="397" alt="Screenshot 2025-07-10 at 6 33 42 PM" src="https://github.com/user-attachments/assets/27351e35-f9fd-4cbb-a7c4-d86c0c2de067" />
With 3 columns right before next break:
<img width="726" height="473" alt="Screenshot 2025-07-10 at 6 36 24 PM" src="https://github.com/user-attachments/assets/d4c43bda-d8cd-445d-8250-d911fbe2f696" />